crates/core/component/distributions/src/component/state_key.rs

```diff
@@ -4,8 +4,8 @@ pub fn staking_token_issuance_for_epoch() -> &'static str {
 }
 
 // The amount of LQT rewards issued for this epoch.
-pub fn lqt_reward_issuance_for_epoch() -> &'static str {
-    "distributions/lqt_reward_issuance_for_epoch"
+pub fn lqt_reward_issuance_for_epoch(epoch_index: u64) -> String {
```

Comment:
Let's create an `lqt` module so that the call site is `lqt::v1::budget::for_epoch` and make it return a `Vec<u8>` since we know we want it to go into NV storage which has a byte oriented keyspace